tfhe/src/shortint/server_key/tests/modulus_switch_compression.rs

```diff
@@ -0,0 +1,39 @@
+use crate::shortint::keycache::KEY_CACHE;
+use crate::shortint::parameters::*;
+use crate::shortint::server_key::tests::parametrized_test::create_parametrized_test;
+use rand::Rng;
+
+const NB_TESTS: usize = 10;
```

Comment:
aren't those constants shared somehow ? Maybe they could be put in the tests/mod.rs wdyt ?

Reply:
Good ides
Moved